### ISSUE TITLE
VBoxManage: LockMachine(a->session, LockType_Write): E_FAIL (0x80004005) - Unspecified error

### ISSUE THREAD

**Issue by SahuMohit** [kubernetes/minikube#5222]
**The exact command to reproduce the issue**:
choco uninstall minikube 
remove ./User/.minikube folder
install minikube 
choco minikube install
minikube start

**The full output of the command that failed**:<details>

</details>

**The output of the `minikube logs` command**:<details>
* minikube v1.3.1 on Microsoft Windows 10 Home Single Language 10.0.17134 Build 17134
* Tip: Use 'minikube start -p <name>' to create a new cluster, or 'minikube delete' to delete this one.
* Starting existing virtualbox VM for "minikube" ...
E0828 18:15:15.171277    9432 start.go:723] StartHost: start: Error setting up host only network on machine start: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe modifyvm minikube --nic2 hostonly --nictype2 virtio --nicpromisc2 deny --hostonlyadapter2 VirtualBox Host-Only Ethernet Adapter #3 --cableconnected2 on failed:
VBoxManage.exe: error: Code E_FAIL (0x80004005) - Unspecified error (extended info not available)
VBoxManage.exe: error: Context: "LockMachine(a->session, LockType_Write)" at line 529 of file VBoxManageModifyVM.cpp
*
X Unable to start VM: start: Error setting up host only network on machine start: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe modifyvm minikube --nic2 hostonly --nictype2 virtio --nicpromisc2 deny --hostonlyadapter2 VirtualBox Host-Only Ethernet Adapter #3 --cableconnected2 on failed:
VBoxManage.exe: error: Code E_FAIL (0x80004005) - Unspecified error (extended info not available)
VBoxManage.exe: error: Context: "LockMachine(a->session, LockType_Write)" at line 529 of file VBoxManageModifyVM.cpp

*
* Sorry that minikube crashed. If this was unexpected, we would love to hear from you:
  - https://github.com/kubernetes/minikube/issues/new/choose

</details>

**The operating system version**:
Windows 10 Home Single Language
Ram - 8 GB
64-bit

**Comment by tstromberg** [comment]
@markcaudill ran into this same issue. I'm baffled by it: in both cases, minikube knows there is a VM that exists. VirtualBox agrees, because there is a locking issue.

However, the VirtualBox driver appears to feel that it is necessary to create a network for the VM. Can either of you share the output of

`minikube start --alsologtostderr -v=8`

That will include the VirtualBox driver log messages. 

I still suspect that `minikube delete` may fix this issue, but would like to get some confirmation.

**Comment by markcaudill** [comment]
I uninstalled and re-installed VirtualBox and removed ~/.minikube (and ~/.minishift as well but probably not related). I believe deleting that directory was the fix because after reinstalling VirtualBox, I re-ran `minikube start` and saw this error which is what led me to delete the `~/.minikube` directory:

```
C:\Users\mark>minikube start
* minikube v1.3.1 on Microsoft Windows 10 Home 10.0.18362 Build 18362
X Unable to load config: open C:\Users\mark\.minikube\profiles\minikube\config.json: Access is denied.
```

After deleting the `.minikube` directory, `minikube start` worked perfectly.

**Comment by tstromberg** [comment]
@author: I believe this issue is now addressed by minikube v1.4, as it now forcibly resets existing VM's on failure. If you still see this issue with minikube v1.4 or higher, please reopen this issue by commenting with `/reopen`

Thank you for reporting this issue!
